src/TurbulenceConvection/EDMF_Updrafts.jl

```diff
@@ -0,0 +1,147 @@
+"""
+Computes tendencies to q_liq and q_ice due to
+condensation, evaporation, deposition and sublimation
```

Comment:
It's confusing that this is called `EDMF_Updrafts.jl`, suggesting it deals with updrafts broadly; however, the scope here is much narrower.